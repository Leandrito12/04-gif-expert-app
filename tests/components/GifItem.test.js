import { GifItem } from "../../src/components/GifItem/GifItem";
import { render } from "@testing-library/react";

describe("testing GiItem", () => {
  const title = "goku";
  const url = "https://aquiles-vaesta.com/xd.jpg";
  test("should match snapshot", () => {
    const { container } = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
});
