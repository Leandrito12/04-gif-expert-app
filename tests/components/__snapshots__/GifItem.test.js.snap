// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing GiItem should match snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="goku"
      src="https://aquiles-vaesta.com/xd.jpg"
    />
    <h4>
      goku
    </h4>
  </div>
</div>
`;
